### ISSUE TITLE
l10n: Suggestion for alternative Italian translations

### ISSUE THREAD

**Issue by nicospor** [nextcloud/server#30943]
I would like to suggest a couple of minor changes to the Italian translations related to the `Dashboard` and `Tasks` apps.
I'm including here a snippet of the changes, since I can't do it directly in Transifex.

The motivation behind these is mainly avoiding user confusion for two reasons:

1. "Cruscotto" is a literal translation of "Dashboard" and is practically never used in web apps / software contexts, so just keeping it as `Dashboard` would be better.
2. `Tasks` is translated as `Attività`, which is the exact same translation as for the `Activity` app. Using `Task` or `Todo` could avoid confusion.

### Dashboard

`apps/dashboard/l10n/it.js:4-5`

```js
"Dashboard" : "Dashboard"
"Dashboard app" : "Dashboard"
```
`apps/dashboard/l10n/it.json:2-3`

```js
"Dashboard" : "Dashboard"
"Dashboard app" : "Dashboard"
```
### Tasks

`apps/tasks/l10n/it.js:11`

```js
"Tasks" : "Task"
```
`apps/tasks/l10n/it.json:9`

```js
"Tasks" : "Task"
```

**Comment by Spartachetto** [comment]
While I do agree that the better translation of "Dashboard" is "Dashboard", I disagree on "Tasks".

Not aware af the usual translation, yet I'd suggest something like "Impegni" or "Da fare".

And sorry because I am aware that the translators use a different to but I do not know it

**Comment by nicospor** [comment]
> Not aware af the usual translation, yet I'd suggest something like "Da fare" (literally "To Do").

Yes, "Da fare" could be a viable translation for "Tasks", thanks.
